core/src/main/java/com/klaytn/caver/kct/kip7/KIP7.java

```diff
@@ -0,0 +1,991 @@
+     * @throws InvocationTargetException
+     * @throws TransactionException
+     */
+    public TransactionReceipt.TransactionReceiptData unpause(SendOptions sendParam) throws NoSuchMethodException, IOException, InstantiationException, ClassNotFoundException, IllegalAccessException, InvocationTargetException, TransactionException {
+        SendOptions sendOptions = determineSendOptions(this, sendParam, FUNCTION_PAUSE, null);
+
+        TransactionReceipt.TransactionReceiptData receiptData = this.getMethod(FUNCTION_UNPAUSE).send(null, sendOptions);
+        return receiptData;
+    }
+
+    /**
+     * Execute a method "unpause" in KIP-7 standard contract.
+     * Caller must have Pauser permission.
+     * It will use default sendOptions in contract instance to passed sendOptions
+     * If a gas value in default sendOptions has null, it will automatically set gas value through estimateGas().
+     * @return TransactionReceipt.TransactionReceiptData
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws ClassNotFoundException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws TransactionException
+     */
+    public TransactionReceipt.TransactionReceiptData renouncePauser() throws NoSuchMethodException, IOException, InstantiationException, ClassNotFoundException, IllegalAccessException, InvocationTargetException, TransactionException {
+        return renouncePauser(this.getDefaultSendOptions());
+    }
+
+    /**
+     * Execute a method "unpause" in KIP-7 standard contract.
+     * Caller must have Pauser permission.
+     * If a gas value in sendOptions has null, it will automatically set gas value through estimateGas().
+     * @param sendParam A SendOptions need to execute contract's method.
+     * @return TransactionReceipt.TransactionReceiptData
+     * @throws NoSuchMethodException
+     * @throws IOException
+     * @throws InstantiationException
+     * @throws ClassNotFoundException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws TransactionException
+     */
+    public TransactionReceipt.TransactionReceiptData renouncePauser(SendOptions sendParam) throws NoSuchMethodException, IOException, InstantiationException, ClassNotFoundException, IllegalAccessException, InvocationTargetException, TransactionException {
+        SendOptions sendOptions = determineSendOptions(this, sendParam, FUNCTION_RENOUNCE_PAUSER, null);
+
+        TransactionReceipt.TransactionReceiptData receiptData = this.getMethod(FUNCTION_RENOUNCE_PAUSER).send(null, sendOptions);
+        return receiptData;
+    }
+
+    private static SendOptions determineSendOptions(KIP7 kip7, SendOptions sendOptions, String functionName, List<Object> argument) throws NoSuchMethodException, IOException, InstantiationException, IllegalAccessException, InvocationTargetException, ClassNotFoundException {
```

Comment:
Can you consider defaultSendOptions together in determineSendOptions function? (for from and gas)